### ISSUE TITLE
BCH stuck in wallet

### ISSUE THREAD

**Issue by jchurchill99** [fyookball/electrum#125]
I downloaded Electron Cash, set up a new wallet using default_wallet. Imported BCC with no problem but cannot send. When I open the default_wallet there is no password and the wallet file is not encrypted, yet when I try to spend the coins it asks me for a password that I do not have. I was sharing this folder with Electrum, could there be a clash? If so, how would I recover the coins?

Thank you

**Comment by fyookball** [comment]
is it a 2fa wallet?

**Comment by jchurchill99** [comment]
It is not.
Thank you
Jamez


Sent from my Galaxy Tab® A
-------- Original message --------From: fyookball <notifications@github.com> Date: 2017-08-14  8:28 AM  (GMT-06:00) To: fyookball/electrum <electrum@noreply.github.com> Cc: jchurchill99 <tdecontrols@gmail.com>, Author <author@noreply.github.com> Subject: Re: [fyookball/electrum] BCH stuck in wallet (#125) 
is it a 2fa wallet?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/fyookball/electrum","title":"fyookball/electrum","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/fyookball/electrum"}},"updates":{"snippets":[{"icon":"PERSON","message":"@fyookball in #125: is it a 2fa wallet?"}],"action":{"name":"View Issue","url":"https://github.com/fyookball/electrum/issues/125#issuecomment-322205218"}}}

**Comment by fyookball** [comment]
im confused then what kind of pwd its asking for. screenshot?

**Comment by coincoinyop** [comment]
Hi, I have he exact same issue yesterday ( 8/14/2017 ) Today I get a message : Transaction was rejected by network rules ( 16: mandatory script flag failed ( signature must use SIGHASH_FORKID) Yesterday it was error 66: about mempool fee ...
I imported an electrum wallet with password and google authenticator code , 2fa on my electrum wallet. I enter them again when asked but transaction keeps failing.
And when I open my electron wallet it doesn't ask for password , it opens right away I don't understand why since when I downloaded it it asked for my password and 2fa .
I want to withdraw my BCC.
Thanks for the help!
![capture error 16](https://user-images.githubusercontent.com/31017624/29314023-0939a884-818a-11e7-892a-65741fd2ec72.PNG)
